src/AvlTree/AvlTreeNode.ts

```diff
@@ -248,8 +248,31 @@ class AvlTreeNode<ValueType> {
     return new Array<number>();
   }
 
-  public findRange(beginValue: ValueType, endValue: ValueType): number[] | null {
-    return new Array<number>();
+  public findRange(beginValue: ValueType, endValue: ValueType): AvlTreeNode<ValueType>[] | null {
```

Comment:
As I've talked to you, this method should return a number array instead of an array of Nodes.

Reply:
Done.